### PR TITLE
Add proposed items to cookiecutter changes

### PR DESCRIPTION
**Pull request by tekktrik**
Using this library as a proxy to record proposed changes to the cookiecutter.  Setting as a draft so it doesn't get merged.  Easier to work on this library than the actual cookiecutter repo for now :)

### CODE REVIEW

**Comment by kattni** [comment]
@tekktrik Thank you for putting this together! Please also change the chat badge URL in the README from:
```
.. image:: https://img.shields.io/discord/327254708534116352.svg
    :target: https://adafru.it/discord
    :alt: Discord
```
To:
```
.. image:: https://github.com/adafruit/Adafruit_CircuitPython_Bundle/blob/main/badges/adafruit_discord.svg
    :target: https://adafru.it/discord
    :alt: Discord
```
(Only the `image` URL is changed, but I included the other bits for context.)

This is something I intend to run as an Adabot patch soon, but is not crucial, so I haven't taken time to do it yet. I'd like to see it in cookiecutter now though.

**Comment by tekktrik** [comment]
@kattni updated!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -42,13 +42,11 @@ jobs:
...
         source actions-ci/install.sh
-    - name: Pip install pylint, Sphinx, pre-commit
+    - name: Pip install Sphinx, pre-commit
       run: |
-        pip install --force-reinstall pylint Sphinx sphinx-rtd-theme pre-commit
+        pip install --force-reinstall Sphinx sphinx-rtd-theme pre-commit
     - name: Library version
       run: git describe --dirty --always --tags
-    - name: Setup problem matchers
-      uses: adafruit/circuitpython-action-library-ci-problem-matchers@v1
```
Comment (jepler) [inline .github/workflows/build.yml:51]: what's the consequence of moving this step?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -18,23 +18,25 @@ repos:
...
         exclude: "^(docs/|examples/|tests/|setup.py$)"
--   repo: local
-    hooks:
-    -   id: pylint_examples
-        name: pylint (examples code)
+    -   id: pylint
+        name: pylint (example code)
         description: Run pylint rules on "examples/*.py" files
-        entry: /usr/bin/env bash -c
-        args: ['([[ ! -d "examples" ]] || for example in $(find . -path "./examples/*.py"); do pylint --disable=missing-docstring,invalid-name,consider-using-f-string $example; done)']
```
Comment (jepler) [inline .pre-commit-config.yaml]: thank you, the fix to how pylint is run is important

File: .github/workflows/build.yml (hunk 1 above)
Comment (tekktrik) [reply .github/workflows/build.yml:51]: I don't really know.  I think what I figured was that it was added to cookiecutter as well as patched, and it just resulted in it being in 2 different places, so I figured picking one as standard would be a good idea.  But to be honest I don't know the significance of this step (as in I don't know what it does haha).

File: .pylintrc [hunk 3]
```diff
@@ -252,7 +252,7 @@ ignore-docstrings=yes
 ignore-imports=yes
 
 # Minimum lines number of a similarity.
-min-similarity-lines=12
+min-similarity-lines=4
```
Comment (kattni) [inline .pylintrc]: Please leave this at 12. We will need to fix the libs as well to return them to 12. Typing adds similar lines to library files, and we ran into this issue during the Sprints, where 4 was no longer sufficient. I do not want to disable the duplicate code check entirely, but returning it to 12 is fine.

**Review by kattni — CHANGES_REQUESTED** [review]
One change below, otherwise this looks good! No idea if you caught everything, but everything here looks valid.

**Comment by tekktrik** [comment]
Okay @kattni, change made!

**Comment by tekktrik** [comment]
I'll use this as the template.

**Comment by tekktrik** [comment]
Changes proposed in this PR for the cookiecutter have been made!  Closing this draft.

**Pull request closed without merging**